client/verta/verta/data_types/_discrete_histogram.py

```diff
@@ -37,6 +36,7 @@ class DiscreteHistogram(_VertaDataType):
 
     _TYPE_NAME = "discreteHistogram"
     _VERSION = "v1"
+    _scipy = maybe_dependency("scipy")
```

Comment:
If it's not too much trouble, for these `maybe_dependency()` calls, I think we can help the user an extra step by putting these in the classes' `__init__()`s with a
```
scipy = maybe_dependency("scipy")
if scipy is None:
    raise ImportError("scipy is not installed; try `pip install scipy`")
```

Otherwise, they'll just see a rather esoteric  `NoneType has no attribute` later on